InvenTree/InvenTree/api.py

```diff
@@ -57,6 +58,18 @@ def get(self, request, *args, **kwargs):
         })
 
 
+class VersionTextView(APIView):
+    """Simple JSON endpoint for InvenTree version text."""
+
+    permission_classes = [
+        permissions.IsAdminUser,
+    ]
+
+    def get(self, request, *args, **kwargs):
+        """Return information about the InvenTree server."""
+        return JsonResponse(inventreeApiText())
```

Comment:
Question: I asked me this a while ago, shouldn't I use the Response from DRF, that automatically is a json response? Or should I use the response from Django is such situations?

Reply:
You can use either